connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2461,6 +2461,60 @@ class Dataset[T] private[sql] (val session: SparkSession, private[sql] val plan:
     new DataFrameWriterV2[T](table, this)
   }
 
+  def unpersist(blocking: Boolean): this.type = {
```

Comment:
Did you skip persist for a reason?

Reply:
persist is implemented in the Python side I believe

Reply:
I don't think it is: https://github.com/apache/spark/blob/master/python/pyspark/sql/connect/dataframe.py#L1509-L1510

Reply:
Oh I now remember:

`def persist(): this.type`
`def persist(newLevel: StorageLevel): this.type`

There is a StorageLevel class in the signature which is from the core module. Does Scala client now can reuse the core 's classes (IIUC scala client is aiming to get rid of core dependency eventually)?

Reply:
Yeah, we can use the storage enum. We will move the enum to common/util later on.